frontend/stores/list-watch.js

```diff
@@ -0,0 +1,41 @@
+import { observable, action } from 'mobx'
+import { createContext } from 'react'
+import axios from 'axios'
+
+class ListWatchStore {
+  @observable watches = []
+
+  @action
+  async fetchWatches(userId) {
+    try {
+      const response = await axios.get('/api/watch-manager/users/' + userId)
+      if (response.status === 200) {
+        this.watches = response.data
+      }
+    } catch (error) {
+      console.error(error)
+    }
+  }
+
+  getWatchById(_id) {
+    return this.watches.find(watch => watch._id === _id)
+  }
+
+  @action
+  async changeWatchStatusById(_id) {
```

Comment:
Khi status chỉ có thể là true hoặc false thôi thì toggle hợp lý hơn.
```suggestion
  async toggleStatus(id) {
```